### DISCUSSION TITLE
Approved but benefits not available.

### DISCUSSION THREAD

**Discussion by yasshraj** [community/community#135685] [GitHub Education]
### Hi everyone,

My request was approved on August 7 but I still have not received my benefits yet. It says it usually takes about 5 days?

**Comment by SabarishCodeWizard** [comment]
how does it work mean github ?

**Comment by davevad93** [comment]
Hi @yasshraj ,

I see you have the `PRO` badge, if the application status label is purple and state "`Congrats! Your request for @yasshraj was approved and your benefits are now available.`" that means you have received your benefits.

**Reply by yasshraj** [reply]
I’m not sure because It says will be available soon but has already been more than 5 days.

**Reply by davevad93** [reply]
Hey @yasshraj , 

If it says "`Congrats! Your request for @yasshraj was approved and your benefits will be available soon.`" that means you still have to wait.

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @yasshraj was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by prabhjotschugh** [comment]
Hi
After approval, it can take about 5-7 days to receive the benefits. You will get an email regarding the same.
Hope it helps!
